src/views/room/index.tsx

```diff
@@ -123,7 +124,7 @@ const Room: FC = () => {
             color={Colors.blue}
             text={t("room.buttons.start")}
             classNames="room__action"
-            disabled={devicesArray.length >= 3 ? false : true}
+            disabled={devicesArray.length >= gameMinPlayers ? false : true}
```

Comment:
Tu pourrais simplifier

```suggestion
            disabled={devicesArray.length < gameMinPlayers}
```

Reply:
carrement